exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/_log_exporter/__init__.py

```diff
@@ -167,8 +167,12 @@ def export(self, batch: Sequence[LogData]) -> LogExportResult:
             resp = self._export(serialized_data)
             # pylint: disable=no-else-return
             if resp.ok:
+                if resp.raw is not None:
```

Comment:
Do you know if moving this right after getting `resp` make `resp.ok` and `self._retryable(resp)` change behavior? If not would be nice to do this just once.